backend/model2_stablediff.py

```diff
@@ -38,13 +38,17 @@ def dummy(images, **kwargs): return images, False
         
         print("---> Stable Diffusion initialized")
 
-
     def generate_images(self, text: str):
+        if config.IMAGE_SEED is None:
+            seed = self.generator.seed()
```

Comment:
At least on my machine this is generating seeds that I can't save into the db: _"Failed to save prompt to db: integer out of range"_ - for example `599035817469872`

Reply:
I haven't tested it with stablediff model - my mistake. 
I have changed the seeds to be in [1, 1024] range: I don't see any value in those huge seeds from pytorch.